common.mk

```diff
@@ -146,29 +146,29 @@ FIRTOOL_TARGETS = \
 	$(FIRTOOL_FILELIST) \
 	$(FIRTOOL_BB_MODS_FILELIST)
 
-ifeq (,$(ENABLE_CUSTOM_FIRRTL_PASS))
-	REPL_SEQ_MEM = none
-	TOP_TARGETS = none
-	HARNESS_TARGETS = none
-	TRANSFORMS = barstools.tapeout.transforms.GenerateTop
-else
-	REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(TOP_SMEMS_CONF)
-	TOP_TARGETS = $(TOP_SMEMS_CONF)
-	HARNESS_CONF_FLAGS = -thconf $(HARNESS_SMEMS_CONF)
-	HARNESS_TARGETS = $(HARNESS_SMEMS_CONF)
-	TRANSFORMS = barstools.tapeout.transforms.GenerateTopAndHarness
-endif
+SFC_REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(SFC_SMEMS_CONF)
 
 
 # DOC include start: FirrtlCompiler
-# hack: lower to middle firrtl if Fixed types are found
-$(HARNESS_TARGETS) $(TOP_TARGETS) $(FIRTOOL_TARGETS) &: $(FIRRTL_FILE) $(FINAL_ANNO_FILE) $(VLOG_SOURCES)
+# This step can take either one of two paths. The first path is when SFC
```

Comment:
This is incorrect. 

1st case: The Chisel generator creates CHIRRTL from the Chisel files. Then FIRTOOL converts this CHIRRTL into Verilog
2nd case: The Chisel generator creates the CHIRRTL from the Chisel files. Then SFC converts CHIRRTL to LowFIRRTL. Then FIRTOOL converts this CHIRRTL into Verilog.

Reply:
Non-blocking Nit: I'm not sure whether we should refer to things as `firtool`, `FIRTOOL`, `CIRCT`, `circt`, etc. Probably worth snooping other sources... but maybe we should call this `MFC` (MLIR FIRRTL compiler) (see https://github.com/sifive/chisel-circt) to match `SFC`